app/services/send_reference_confirmation_email.rb

```diff
@@ -0,0 +1,9 @@
+class SendReferenceConfirmationEmail
+  def self.call(application_form:, reference:)
+    RefereeMailer.reference_confirmation_email(application_form, reference).deliver
+
+    audit_comment = I18n.t('reference_confirmation_email.audit_comment', referee_name: reference.name, referee_email: reference.email_address)
+    application_comment = SupportInterface::ApplicationCommentForm.new(comment: audit_comment)
+    application_comment.save(application_form)
```

Comment:
What would you think of using this approach?

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/99f0f2fd1a01c42457b1b13a2309fe26825ff224/app/services/send_chase_email_to_provider.rb#L11-L16

- Use the object directly so we don't have to use `SupportInterface` code in the top-level
- Perhaps skip i18n for this, might make it easier to read

Reply:
Done!